CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does not yet adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 for setuptools_scm/PEP 440 reasons.
 
+
+## UNRELEASED
+
+### Changed
+
+### Fixed
+
+- Fixed excess memory use when displaying plot logs in GUI
```

Comment:
I was under the impression that the GUI would request the logs regardless of whether it was showing them or not